src/amperity/gocd/agent/aurora/agent.clj

```diff
@@ -162,7 +173,7 @@
 
 (comment
   {:state :running
-   :environment "build"
+   :environments "build,test"
```

Comment:
Since this is internal state, could this be a set or vector of envs instead?

Reply:
I'm going to push back on this because I think it's easier to reason about if it's always a string, instead of having remember when it's a string vs when it's a set. It's not really internal state in that it's managed by GoCD and we don't have a single point in the API where all interactions with the state go through where we can cleanly transform the state.

Reply:
Makes sense to me. 👍